### PR TITLE
Qcdigit task

### PR DESCRIPTION
**Pull request by tmtetwa**

### CODE REVIEW

File: Modules/TRD/include/TRD/LinkDef.h [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#ifdef __CLING__
+#pragma link off all globals;
+#pragma link off all classes;
+#pragma link off all functions;
+
+#pragma link C++ class o2::quality_control_modules::trd::TRDDigitQcTask+;
```
Comment (tdietel) [inline Modules/TRD/include/TRD/LinkDef.h:6]: I would rename this class to `o2::quality_control_modules::trd::DigitsTask`. This seems to be more in line with TOF and TPC naming conventions.

File: Modules/TRD/test/testQcTRD.cxx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+///
+/// \file   testTRD.cxx
```
Comment (tdietel) [inline Modules/TRD/test/testQcTRD.cxx:12]: Needs cleanup.

File: Modules/TRD/TRDDigitQcTask.json [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+        "taskParameters": {
+          "trdDigits": "myOwnValue"
+        },
+        "location": "remote"
+      }
+    },
+    "checks": {
+      "QcCheck": {
+        "active": "true",
+        "className": "o2::quality_control_modules::trd::SimpleTrdCheck",
```
Comment (tdietel) [inline Modules/TRD/TRDDigitQcTask.json:47]: Is this test already available?

Comment (tmtetwa) [reply Modules/TRD/TRDDigitQcTask.json]: It wasn't I just recently updated it

File: Modules/TRD/test/testQcTRD.cxx (hunk 2 above)
Comment (tmtetwa) [reply Modules/TRD/test/testQcTRD.cxx]: Okay, I just removed those lines

**Review by Barthelemy — APPROVED** [review]
All good

**Comment by Barthelemy** [comment]
Warnings are now considered errors. Could you just fix the couple of warnings we see in the builders logs ? thank you

**Review by tdietel — APPROVED** [review]
Looks good.

**Comment by Barthelemy** [comment]
test failure is unrelated. Merging.

**Pull request merged**